spec/controllers/tasks_controller_spec.rb

```diff
@@ -154,6 +155,15 @@
         expect(assigns(:task).license).to match(valid_params[:license_id])
       end
 
+      it 'sets labels correctly' do
+        post_request
+        expect(assigns(:task).labels.map(&:name)).to eq(valid_params[:label_names])
```

Comment:
The `eq` matcher will care about the order of elements in this array. If you ensure a stable sorting: Keep it. If you don't, you might also switch to another array matcher as outlined [here](https://medium.com/beambenefits/array-matchers-in-rspec-42b370e3983e).

Reply:
i switched to contain_exactly because the order is not ensured to stay the same.